pcap/pcap.h

```diff
@@ -825,12 +825,12 @@ PCAP_API const char *pcap_lib_version(void);
   typedef struct pcap_send_queue pcap_send_queue;
 
   /*!
-    \brief This typedef is a support for the pcap_get_airpcap_handle() function
-  */
-  #if !defined(AIRPCAP_HANDLE__EAE405F5_0171_9592_B3C2_C19EC426AD34__DEFINED_)
-    #define AIRPCAP_HANDLE__EAE405F5_0171_9592_B3C2_C19EC426AD34__DEFINED_
-    typedef struct _AirpcapHandle *PAirpcapHandle;
-  #endif
+  \brief This typedef is a support for the pcap_get_airpcap_handle() function
```

Comment:
I'd leave the indentation alone, to indicate that this is only done if `_WIN32` is defined, and keep the `*` styling the same.